application/src/main/java/org/togetherjava/tjbot/commands/generic/DatabaseCommand.java

```diff
@@ -31,43 +31,41 @@
  * }
  * </pre>
  */
-public final class DatabaseListener extends ListenerAdapter {
-
-    private static final Logger logger = LoggerFactory.getLogger(DatabaseListener.class);
-
+public class DatabaseCommand extends AbstractCommand {
+    private static final Logger logger = LoggerFactory.getLogger(DatabaseCommand.class);
     private final Database database;
 
     /**
      * Creates a new command listener, using the given database.
      *
      * @param database the database to store the key-value pairs in
      */
-    public DatabaseListener(Database database) {
+    public DatabaseCommand(Database database) {
+        super("Database Get & Set",
+                "This command can set or retrieve values from a key-object database.", true);
         this.database = database;
     }
 
```

Comment:
Override the Command#getOptions to add a "key" and "value" option. (And to add subcommands)